### ISSUE TITLE
file lock is null

### ISSUE THREAD

**Issue by jamieb22** [orientechnologies/orientdb#7095]
Not sure if this is a debug or not. Seeing debug errors.

## OrientDB Version, operating system, or hardware.
- v2.2.15

## Operating System
- [ X] Linux

## Expected behavior and actual behavior

2017-01-18 15:46:11.707 INFO  - - shutdown storage: archiva$lf...
2017-01-18 15:46:11.754 ERROR - File lock is null , file lock is not released
```

@Override
	public void close() throws DatabaseException {
		if (connected.compareAndSet(true, false)) {
		    OStorage storage = databaseUtils.getGraphFactory().getDatabase().getStorage();
		    storage.close(true,false);
			databaseUtils.getGraphFactory().getDatabase().close();
			databaseUtils.getGraphFactory().close();
		}
	}
```

**Comment by laa** [comment]
Fixed yesterday

**Comment by jamieb22** [comment]
Did the fix make it into 2.2.15? I don't see the commit in 2.2.x branch.

**Comment by laa** [comment]
No it is 2.2.16
